types/react-plotly.js/index.d.ts

```diff
@@ -94,7 +95,7 @@ export interface PlotParams {
     onTransitioning?: () => void;
     onTransitionInterrupted?: () => void;
     onUnhover?: (event: Readonly<Plotly.PlotMouseEvent>) => void;
+    onBeforeHover?: (event: Readonly<Plotly.PlotMouseEvent>) => boolean;
```

Comment:
This list is in alphabetical order to match the source package's readme. Please maintain that with your change.

Reply:
Thanks for the feedback. I didn't realize that the list was alphabetical. I made the change.